ui/src/message_popup/mobile_prototype/mobile_interface.jsx

```diff
@@ -0,0 +1,383 @@
+    return (
+      <div>
+        <AppBar
+          style={styles.appBar}
+          title="Message PopUp Mobile"
+          iconElementLeft={
+            <IconButton
+              onTouchTap={this.resetExperience}
+            >
+              {!hasStarted &&
+                <MenuIcon 
+                  color={fullWhite}
+                />
+              }
+              {hasStarted && questionsAnswered < sessionLength &&
+                <ArrowBackIcon 
+                  color={fullWhite}
+                />
+              }
+              {hasStarted && questionsAnswered >= sessionLength &&
+                <RefreshIcon 
+                  color={fullWhite}
+                />
+              }
+            </IconButton>
+          }
+          />
+        <div>
+        {/*Header*/}
+          
+        </div>
+        <div>
+        {/*Scollable Body*/}
+          {!hasStarted && 
+            <div style={styles.instructionsBody}>  
+              <InstructionsCard 
+                onStartPressed={this.onStart}
+                email={email}
+                itemsToShow={query}
+                />
+            </div>
+          }
+          {hasStarted && questionsAnswered < sessionLength &&
+            <div>
+              <LinearProgress style={styles.progressBar} color="#EC407A" mode="determinate" value={questionsAnswered} max={sessionLength} />
+              <div style={styles.textBody}>
+                <TextBody 
+                  messages={this.state.messages}
+                  />
+              </div>
+            </div>
+          }
+          {hasStarted && questionsAnswered >= sessionLength &&
+            <div style={styles.doneBody}>
+              <div style={styles.doneBodyText}>
+                You finished!
+              </div>
+              <Divider />
+              <FinalSummaryCard 
+                responseTimes={this.props.responseTimes}
+                limitMs={this.props.limitMs}
+              />
+              <RaisedButton 
+                label="Done"
+                style={styles.doneBodyButton}
+                secondary={true}
+              />
+            </div>
+          }
+        </div>
+        <div>
+        {/*Footer*/}
+          {hasStarted && questionsAnswered < sessionLength &&
+            <div style={styles.textFooter}>
+              <TextFooter 
+                onQuestionDone={this.onNextQuestion}
+                addResponse={this.addResponse}
+                isReadyToMoveOn={this.isReadyToMoveOn}
+                helpType={this.props.helpType}
+                elapsedMs={this.state.elapsedMs}
+                showExample={this.showExample}
+                nextExample={this.nextExample}
+                nextButtonLabel={this.props.isLastQuestion ? 'Finish' : 'Next Question'}
+                />
+            </div>
+          }
+        </div>
+      </div>
+    );
+  }
+});
+
+
+const styles = {
+  appBar: {
```

Comment:
Do these all need to be absolutely positioned?  We should try to do some flexbox-ifying :)  I can pair with you on that too if you like.